ReSharper.FSharp/src/FSharp/FSharp.Psi.Intentions/src/Intentions/FunctionAnnotationAction.fs

```diff
@@ -162,42 +207,72 @@ module SpecifyTypes =
             | :? ITuplePat as tuplePat -> addParens factory tuplePat
             | pattern -> pattern
 
+        let displayContext = displayContext.WithShortTypeNames(true)
```

Comment:
Any reason to do it repeatedly for all the patterns instead of the calling code?